### DISCUSSION TITLE
Story River (similar to Tiddly Wiki)

### DISCUSSION THREAD

**Discussion by j-steinbach** [srid/emanote#140] [Ideas]
One (maybe *the*) feature of [Tiddly Wiki](https://tiddlywiki.com/) (TW) is their story river. Opening a link inside the TW doesn't close the current note, but instead appends the new note to the current note, then navigates to it. That means the context *where* the note was opened is kept. 

- A slightly different approach is taken [in this TW](http://tw5.scholars.tiddlyspot.com/#William%20Shakespeare). The river doesn't flow vertically but instead is implemented as a stack. (Click on any of the links, then click the X in the corner. See how it takes you back to your previous note?)

This concept is very useful for a Zettelkasten (ZK) approach, as the ZK can be thought of as a second brain, with which we correspond (have a dialogue/discussion). Context _is_ important. 

Taking a step back, the [Folgezettel hierarchy](https://neuron.zettel.page/folgezettel) can be seen as a form of a story river, except it doesn't show the full notes, but instead their ids/headings.

**Comment by vcavallo** [comment]
Oooo, that is _nice_. that would bring a whole new layer of approach for building up the context necessary to create good structure notes and concept-joining notes.
